static/css/style.css

```diff
@@ -0,0 +1,13 @@
+nav {
```

Comment:
Oh sorry, just realised we already have a `main.css` we should keep this there. Sorry 😓